validator.py

```diff
@@ -1,22 +1,16 @@
 import sys
 from os import path, getcwd
 
-from colors import color
```

Comment:
What happened to the colors? :p

Reply:
You don't see the colours then this is running in the depths of testing. The idea is to get a result, true or false.